libs/ui/design-system/src/lib/textarea.tsx

```diff
@@ -1,8 +1,23 @@
 import { Label } from '@nmit-coursition/ui/primitives/label'
-import { Textarea as TextareaPrimitive } from '@nmit-coursition/ui/primitives/textarea'
+import * as TextareaPrimitive from '@nmit-coursition/ui/primitives/textarea'
 import { cn } from '@nmit-coursition/ui/utils'
+import { type VariantProps, cva } from 'class-variance-authority'
 
-interface TextareaWithTextProps extends React.ComponentProps<'textarea'> {
+const textAreaVariants = cva('', {
+  variants: {
+    variant: {
+      primary: '',
+      invalid: '',
+      outlined: '',
```

Comment:
**question:** Where is outlined defined? It's nowhere in props in Figma. TextArea has no variants, so it doesn't need CVA at all. :)

![image](https://github.com/user-attachments/assets/5e066b25-e205-4967-a40c-d623bcea6cb3)